extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -495,8 +505,12 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       this.extensions_./*OK*/installExtensionForDoc(
           this.getAmpDoc(), 'amp-analytics');
     }
-    this.fireDelayedImpressions(responseHeaders.get('X-AmpImps'));
-    this.fireDelayedImpressions(responseHeaders.get('X-AmpRSImps'), true);
+    if (!this.isFluid) {
+      this.fireDelayedImpressions(responseHeaders.get('X-AmpImps'));
```

Comment:
Are these new header names, or existing ones?

Reply:
These are new, and should be in production before this change is.

Reply:
Is it too late to use a different naming convention? The `X-` convention is no longer recommended, and CamelCase makes little sense on headers since they're case insensitive.

Reply:
You'll have to take that up with @keithwrightbos, he added the header.